clients/client-python/build.gradle.kts

```diff
@@ -21,11 +22,29 @@ fun deleteCacheDir(targetDir: String) {
   }
 }
 
+fun waitForPort(port: Int, host: String = "localhost", timeout: Long = 60000) {
+  val startTime = System.currentTimeMillis()
+  while (true) {
+    try {
+      Socket(host, port).use { return }  // If this succeeds, the port is open
+    } catch (e: Exception) {
+      // Port is not open yet, continue to wait
+    }
+    if (System.currentTimeMillis() - startTime > timeout) {
+      throw RuntimeException("Timed out waiting for port $port to be available")
+    }
+    Thread.sleep(1000)  // Wait for 1 second before checking again
+  }
+}
+
 fun gravitinoServer(operation: String) {
     val process = ProcessBuilder("${project.rootDir.path}/distribution/package/bin/gravitino.sh", operation).start()
     val exitCode = process.waitFor()
     if (exitCode == 0) {
       val currentContext = process.inputStream.bufferedReader().readText()
+      if (operation == "start") {
+        waitForPort(8090)
```

Comment:
Do we need to try to catch `throw RuntimeException("Timed out waiting for port $port to be available")` in here?
```
try {
   waitForPort(8090)
} 
catch Exception()....

```

Reply:
Let me see

Reply:
I have added the exception from lines 33 to 34.